tools/i18n-temp/ru.ts

```diff
@@ -1,60 +1,62 @@
 export const i18n = {
-  tower: 'tower',
-  wall: 'wall',
-  resource: 'resource',
+  tower: 'Башня',
+  wall: 'Стена',
+  resource: 'Ресурс', // where is this used? couldn't find
 
-  bricks: 'кирпичи',
-  gems: 'самоцветы',
-  recruits: 'звери',
+  bricks: 'Камень', 
+  gems: 'Самоцветы',
+  recruits: 'Существа', 
 
-  quarry: 'quarry', // brick production
-  magic: 'magic', // gem production
-  dungeon: 'dungeon', // recruit production
+  quarry: 'Карьер', 
+  magic: 'Магия',
+  dungeon: 'Подземелье', // i wonder what kind of recruits you can get in a dungeon ;)
 
   discarded: 'сброшена',
   'Discard a card': 'Сбросить карту',
 
-  'You Win!': 'You Win!',
-  'You Lose!': 'You Lose!',
-  'Tie Game': 'Tie Game',
+  'You Win!': 'Вы победили!',
+  'You Lose!': 'Вы проиграли!',
+  'Tie Game': 'Ничья!',
 
-  Preferences: 'Preferences',
-  'Your Name:': 'Your Name:',
-  "Opponent's Name:": "Opponent's Name:",
+  Preferences: 'Настройки',
+  'Your Name:': 'Ваше имя:',
+  "Opponent's Name:": "Имя противника:",
   'Choose a Tavern (Preset Preferences):':
-    'Choose a Tavern (Preset Preferences):',
-  Default: 'Default',
-  Customized: 'Customized',
-  'Starting Conditions': 'Starting Conditions',
-  'Victory Conditions': 'Victory Conditions',
-  'Other Preferences': 'Other Preferences',
-  'Cards in Hand': 'Cards in Hand',
-  'AI Type': 'AI Type',
-  'Apply & New Game': 'Apply & New Game',
-  Cancel: 'Cancel',
-
-  Language: 'Language',
-  Volume: 'Volume',
-  Mute: 'Mute',
-  Help: 'Help',
-  'Toggle Full Screen': 'Toggle Full Screen',
-
-  ERATHIAN: 'Use Erathian [%s] (for latin letters only)',
-
-  'ArcoMage HD': 'ArcoMage HD',
+    'Выберите Таверну (набор настроек):',
+  Default: 'По умолчанию',
+  Customized: 'Пользовательская',
+  'Starting Conditions': 'Стартовые условия',
+  'Victory Conditions': 'Условия победы',
+  'Other Preferences': 'Другие настройки',
+  'Cards in Hand': 'Карт в руке',
+  'AI Type': 'Тип ИИ', // couldn't find ingame
+  'Apply & New Game': 'Применить и начать игру', // 7 symbols longer, may want to use just "Начать игру!" instead if the button doesn't stretch
+  Cancel: 'Отмена',
+
+  Language: 'Язык',
+  Volume: 'Громкость',
+  Mute: 'Заглушить',
+  Help: 'Помощь',
+  'Toggle Full Screen': 'Полноэкранный режим',
+
+  ERATHIAN: 'Использовать Эрафийский [%s] (только для латиницы)', //probably not needed in RU locale at all
```

Comment:
It's actually needed in all langs, espacially in non-latin-based langs because it serves as a notification for non latin